lib/routes/bookings/driver/routes.js

```diff
@@ -0,0 +1,47 @@
+import get from 'lodash/get';
+import { notFound, badImplementation } from 'utils/responseInterceptors';
+import { findAllBookingsForDriverId } from 'daos/bookingsDao';
+import { transformDbArrayResponseToRawResponse } from 'utils/transformerUtils';
+
+module.exports = [
+    {
+        method: 'GET',
+        path: '/{driverId}',
+        handler: async (request, h) => {
+            const driverId = request.params.driverId;
+            const { limit } = request.query;
+            return findAllBookingsForDriverId(limit, driverId)
+                .then(bookings => {
+                    if (get(bookings.allBookings, 'length')) {
```

Comment:
Why are we using get here any reason ?

Reply:
Yes, if `allBookings` is empty we could return `notFound` for the given id, otherwise code would be unreachable.